README.md

```diff
@@ -108,20 +108,24 @@ Recomputes embeddings.
 | `feature_distance_metric` | string | Optional  | `'cosine'` | Metric used for calculating feature distance. Options include `cosine` and `euclidean`. Refer to [torch-re-id model zoo](https://kaiyangzhou.github.io/deep-person-reid/MODEL_ZOO.html) to select the metric that matches your model.|
 | `cooldown_period_s`       | float  | Optional  | `5`           | Duration for which the trigger is on.`new_object_detected`.                                          |
 | `re_id_threshold`         | float  | Optional  | `0.3`         | Threshold for determining whether two persons match based on body features similarity.    |
-| `min_track_persistence`   | int    | Optional  | `10`          | Minimum number of frames a track candidate must persist before beinfg promoted to a track.                         |
+| `min_track_persistence`   | int    | Optional  | `4`          | Minimum number of frames a track candidate must persist before beinfg promoted to a track.                         |
 | `max_frequency_hz`           | float  | Optional  | `10`          | Frequency at which the tracking steps are performed. |
 | `save_to_db`              | bool   | Optional  | `True`        | Indicates whether tracks should be saved to the database.                       |
 | `save_period`      | int    | Optional     | `20`    | Interval (in number of tracking steps) when tracks are saved to the database.               |
 | `start_fresh`             | bool   | Optional  | `False`       | Whether or not to load the tracks from the database at `reconfigure()`.                             |
 | `path_to_known_persons`   | string | Optional  | `None`        | Path to the database containing pictures of entire persons. If the directory does not exist it will be created at `reconfigure()`. Refer [example directory tree](#example-of-directory-tree) to see how to add pictures of known persons and associate labels with the persons.            |
 
+
 ### Person detector attributes
 
 | Name                              | Type   | Inclusion    | Default                                 | Description                                                                                                     |
 | --------------------------------- | ------ | ------------ | --------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
 | `detector_model_name`             | string | Optional| `'fasterrcnn_mobilenet_v3_large_fpn'` | Name of the model used for detection. Only option at the moment. Options include `'fasterrcnn_mobilenet_v3_large_320_fpn'` (low resolution)  and `'fasterrcnn_mobilenet_v3_large_fpn'` (high resolution)  |
 | `detection_threshold`             | float  | Optional     | `0.9`                                   | Confidence threshold for detecting objects, with values ranging from 0.0 to 1.0.                                |
```

Comment:
do you want this to increase to 0.92 too? Or rather, once you detect some false positives, see what their confidence is, and raise it to be above those